packages/cdk/lib/transcription-service.ts

```diff
@@ -70,6 +71,13 @@ export class TranscriptionService extends GuStack {
 			{
 				app: APP_NAME,
 				bucketName: `transcription-service-source-media-${this.stage.toLowerCase()}`,
+				cors: [
+					{
+						allowedOrigins: [`https://${domainName}`],
```

Comment:
Does this not affect the worker getObhect call?

Reply:
I don't think it will. CORS rules control which origin urls browsers can access the bucket from (and allowed headers, methods, etc.).
I don't think requests from other backend services should be affected.